app/assets/stylesheets/config/_colors.scss

```diff
@@ -69,9 +69,9 @@
   --accent-warning-a10: #{rgba(#ffcf4c, 0.1)};
 
   // Danger
-  --accent-danger: #dc1818;
-  --accent-danger-darker: #c20a0a;
-  --accent-danger-lighter: #ec5050;
+  --accent-danger: #d31616;
```

Comment:
You're not going to enjoy this but...

It turns out this change means that the 'primary destructive' (i.e. red filled with the light text on it) button doesn't have sufficient color contrast 🙈  Can we use the `--accent-darker` variable for the filled button background color maybe?

Reply:
Okeeey.. It was either not enough coffee or too much coffee for me :D 

Anyway, I've updated PR with slightly changed strategy for colors. I've introduced few CSS variables in main colors file which will be a kickoff for bigger variables overhaul. More info in code comment. Nevertheless, it should now meet the contrast ratio requirements.